new-content/events/summer-school-2021-page/faq.md

```diff
@@ -17,6 +17,9 @@ linearalgebra:
 qceducators:
   url: https://quantum-computing.ibm.com/programs/educators
   segment: quantum-computing-educators
+mailtoQcEducators:
+  url: mailto:quantum.education@us.ibm.com
+  segment: mailto-quantum-educators
```

Comment:
We are trying to unify how we do things, so, until we find a better way to do it, please, follow here the same structure as we have on `summer-school-2021.yaml`:

```
segment:
      action: 'events-summer-school > faq > mailto-quantum-educators'
```

Same for all the links. And please, remove the code you don't need